lib/galaxy/managers/hdas.py

```diff
@@ -347,6 +347,7 @@ def __init__(self, app: StructuredApp):
                 "url",
                 "create_time",
                 "update_time",
+                "creating_job",
```

Comment:
Did we come up with a good way to add the cost of this to the summary?  I recall us thinking we could just drop this part of the old approach and not require creating_job if we relied on checking the active IT store instead.

Reply:
If the hda does not know what job created it then I am not sure how to match it with an IT. I guess we could approach it from the other side and match from some extra properties on the IT object?

Reply:
Right, I think we can add whatever extra info to the IT endpoint if need be (just directly include the hda id?), then cross reference in contentItem render.

Reply:
If we can do that that'd be 💯! I am not totally opposed to including the job id in the summary response (we might need it anyway if we want to have a job centric view), but the performance impact is unclear, and it was a pain to fix the performance issues for the new history. Down the road we could make sure we only include the job id column from the dataset table and bypass the python decorator, paired with some solid automated performance testing so we can make informed decisions.

Reply:
> match from some extra properties on the IT object

I've done this in 266786d -- definitely cheaper for now albeit dirtier. Thanks for the feedback!